Makefile

```diff
@@ -80,29 +89,29 @@ clean-dist:
 
 .PHONY: check
 check: pyenv buildhash
-	@set -e && \
+	set -e && \
 	for dir in $(CHECKABLE_RS); do \
 	  $(SUBMAKE) -C $$dir check; \
 	done; \
-	. pyenv/bin/activate && \
+	. pyenv/$(if ${IS_WINDOWS},Scripts,bin)/activate && \
 	$(SUBMAKE) -C rspy develop && \
 	$(SUBMAKE) -C pylib develop && \
 	for dir in $(CHECKABLE_PY); do \
 	  $(SUBMAKE) -C $$dir check; \
 	done;
-	@echo
-	@echo "All checks passed!"
+	echo
+	echo "All checks passed!"
```

Comment:
Removing the @ means the text is printed twice on other platforms. Could you find a better way?

Reply:
Ops. You are right! I should not remove those.

Reply:
There are still a bunch of lines with @ removed - any reason for that?